src/theme/foundations/colors.ts

```diff
@@ -0,0 +1,25 @@
+export const colors = {
```

Comment:
Are these meant to be used with design system?

Reply:
Hmm currently these are copied over from our existing template! Not sure if we're planning to revamp isomer template colours as well

Reply:
Got it, yea we should maintain current colors. Was just checking if this is compatible with design system or it expects in a different type structure but guess we can figure that later

Maybe we should put into a folder structure. This can be called as Isomer classic

In case we want to "load" a different theme in future we can create a new folder with "modern/theme.ts for example

Reply:
sure, that makes sense! moved in https://github.com/isomerpages/isomer-components/pull/2/commits/bbb8f1c2e1d8425dcd27d9116873df7f5f18d692